### ISSUE TITLE
question

### ISSUE THREAD

**Issue by zhouyuan888888** [aasharma90/LightWeight_RefineNet#4]
In the official code, I only can get 73.1% mIoU on the validation set of cityscapes by setting CropSize=800 and ShortSide=1024 the same as you. Although the same configuration script, I
sincerely achieved around 75% mIoU in the script you provided. Can you give me some suggestion about the most possible reason for this phenomenon and what the modification brings about 2% improvement in your scripts。
 thank you for your excellent work 0（*o*）0 ！！

**Comment by aasharma90** [comment]
Hi @zhouyuan888888, 

Well, I don't think there is anything specific I can think of that might have contributed to the increment. Actually, the numbers reported by segmentation networks are prone to fluctuation. You can see the link below-
https://github.com/speedinghzl/pytorch-segmentation-toolbox/issues/23

So, it is always better to run the training process 3-5 times, and then record the mean value. The score that I obtained, 75.28%, is based only a single trial. So, it is difficult to say anything conclusively. Also, thank you for your appreciation.

**Comment by zhouyuan888888** [comment]
Thank you for your replying 0（*——*）0
